Core/GDCore/Extensions/Builtin/SceneExtension.cpp

```diff
@@ -136,6 +136,20 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsSceneExtension(
       .AddCodeOnlyParameter("currentScene", "")
       .AddParameter("yesorno", _("Deactivate input when focus is lost"))
       .MarkAsAdvanced();
+
+  extension
+      .AddCondition(
+          "HasGameJustResumed",
+          _("Game resumed"),
```

Comment:
Maybe we should have the "just" here:

```suggestion
          _("Game has just resumed"),
```